website/static/js/profile.js

```diff
@@ -38,7 +39,8 @@ var SerializeMixin = function() {};
 
 /** Serialize to a JS Object. */
 SerializeMixin.prototype.serialize = function() {
-    return ko.toJS(this);
+    var self = this;
```

Comment:
What's the purpose of defining `self` in most of these functions?

I understand that it's useful in instances where you're supplying an anonymous function as a callback to an AJAX request (because that function is bound to the response, not the viewmodel, when called) - but in this case in particular it seems to not add anything.